homeassistant/components/prana/quality_scale.yaml

```
@@ -0,0 +1,64 @@
+rules:
+  # Bronze (must be satisfied or exempt)
+  action-setup:
+    status: exempt
+    comment: Integration registers no custom services/actions
+  appropriate-polling: done # coordinator with 10s local interval (>5s min)
+  brands:
+    status: exempt
+    comment: Brand assets will be contributed separately
+  common-modules: done # uses DataUpdateCoordinator pattern
+  config-flow-test-coverage: done # tests exercise zeroconf + confirm paths
+  config-flow: done
+  dependency-transparency: done # no external requirements declared
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done # entities subscribe via async_added_to_hass
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done # coordinator stored in ConfigEntry.runtime_data
+  test-before-configure: done # config flow validates discovery metadata
+  test-before-setup: done # coordinator refreshed prior to platform setup
+  unique-config-entry: done # unique_id based on mdns name prevents duplicates
+
+  # Silver (future work)
+  action-exceptions: todo
+  config-entry-unloading: done # async_unload_entry implemented
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: todo
+  integration-owner: done # codeowners present
+  log-when-unavailable: todo
```

```suggestion
  log-when-unavailable: done
```
Coordinator does that